diff -Naur a/gcc/config.gcc b/gcc/config.gcc
--- a/gcc/config.gcc	2019-08-03 20:16:22.000000000 +0000
+++ b/gcc/config.gcc	2019-08-22 22:36:08.943288147 +0000
@@ -1189,6 +1189,13 @@
 	    tmake_file="$tmake_file arm/t-linux-androideabi"
 	    ;;
 	esac
+	case ${target} in
+	arm*-*-*eabihf)
+	    with_cpu=${with_cpu:-cortex-a53}
+	    with_fpu=${with_fpu:-neon-fp-armv8}
+	    with_float=${with_float:-hard}
+	    ;;
+	esac
 	# The EABI requires the use of __cxa_atexit.
 	default_use_cxa_atexit=yes
 	with_tls=${with_tls:-gnu}
